service/docs/Makefile.mk

```
@@ -48,6 +48,7 @@ EXTRA_DIST += docs/geninfo.sh \
               docs/source/controls_SKX.rst \
               docs/source/devel.rst \
               docs/source/index.rst \
+              docs/source/istall.rst \
```

```suggestion
              docs/source/install.rst \
```

Shouldn't this break the build?